src/Controller/HomepageController.php

```diff
@@ -17,6 +18,9 @@ public function homepageAction()
     {
         // FIXME: Récupérer les utilisateurs non admin
         $users = [];
+        $repo = $this->getDoctrine()->getRepository(User::class);
```

Comment:
Ici pas la peine de faire une variable pour le repository. Tu peux directement faire `$users = $this->getDoctrine()->getRepository(User::class)->findBy(['isAdmin' => false])`